tools/local-test/scripts/lib/print.sh

```
@@ -1,16 +1,35 @@
-RED='\033[0;31m'
-GREEN='\033[0;32m'
-YELLOW='\033[1;33m'
+RED=$(tput setaf 1)
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)